src/aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -582,7 +582,6 @@ async def execute(self) -> plumpy.process_states.State:  # type: ignore[override
         except TransportTaskException as exception:
             raise plumpy.process_states.PauseInterruption(f'Pausing after failed transport task: {exception}')
         except plumpy.process_states.KillInterruption as exception:
-            await self._kill_job(node, transport_queue)
```

Comment:
So we do not trigger in the ProcessState anymore in the cancelation of the scheduler job. This made it hard to manage the kill command as killing happens at two places.